functions/src/index.ts

```diff
@@ -1,27 +1,46 @@
-import {config, firestore as database, https} from 'firebase-functions';
+import {config, firestore as firestoreFromFunctions, https} from 'firebase-functions';
```

Comment:
you can just name it "firestore"

Reply:
There's another import called `firestore`.
So i've changed this to be firestore and renamed the other one firestoreAdmin